plugin/src/types.ts

```diff
@@ -6,6 +6,7 @@ export interface ReanimatedPluginPass {
   opts: {
     relativeSourceLocation?: boolean;
     disableInlineStylesWarning?: boolean;
+    useOnExitLogicForWorkletNodes?: boolean;
```

Comment:
Good names describe what is done, not how. Please consider renaming this option to something like `processNestedWorklets` or similar.